fastlane/Fastfile

```diff
@@ -124,6 +124,12 @@ platform :ios do
     check_no_github_release_exists(version_number)
   end
 
+  desc "tvOS and watchOS build"
+  lane :build_tv_and_watch do |options|
+    check_pods
+    carthage(command: "build", no_skip_current: true, platform: "watchOS,tvOS", use_xcframeworks: true)
```

Comment:
I just realized we can safely go back to this syntax, now, awesome! 
we have a couple of carthage commands we were calling through `sh` because a carthage bug prevented us from calling `carthage` directly, and we used a script instead. We've since removed the script, but didn't move back to this nicer syntax in `carthage_archive`

Reply:
if this starts to run long (because carthage starts building all the schemes it can find in the repo), we can replace it with `fastlane gym` and pass it the schemes we need